### DISCUSSION TITLE
Problems Updating Felamimail

### DISCUSSION THREAD

**Discussion by selfarian** [tine-groupware/tine#39] [General]
Hello, 
I had a very old version of tine running, because of some issues with my MySQL Server. So, some time ago, I updated my SQL Server and tried to update tine. 
First I tried to update from "old" to the "newest" version, what didn't work. So I tried smaller steps. I am unsure, to which version I try to update, I can only say, that the issue is with Felamimail updating (with setup) from 15.4 to 15.6. When I try to do this, I get the following error:
Tinebase_Server_Json::_handleException::443 Zend_Db_Statement_Exception -> SQLSTATE[42S02]: Base table or view not found: 1146 Table 'xxxxx_mail.tine20_path' doesn't exist, query was: TRUNCATE TABLE `tine20_path`

Which is correct, because there is no table "tine20_path"... 
Can someone please help me? :)

Thank you and best regards,
Alex

**Comment by lab-at-nohl** [comment]
I cannot find any mention of pathes between 15.4 and 15.6 of Felamimail. However, a lot of Truncates are made to empty mail cache. Maybe there are dependencies (the query is really seldom). Try to empty Felamimail cache tables manually (Backup!!). Do you have access to the cli (SSH)?

Given the simple query, you could also try to add the missing table.

**Comment by selfarian** [comment]
Hi, thanks for your answer.
Can you tell me, which tables i should empty?

I have full access to the server, so ssh, mysql, phpmyadmin... whatever :)

I thought about creating the table, too, but I didn't find it's structure. So maybe you have a hint for me?

**Comment by lab-at-nohl** [comment]
For the table structure see here: https://github.com/tine20/tine20/issues/7440

Also you can disable featureSearchPath. I will show later if above does not work. 

Last, you can try to empty all felamimail_cache_* tables, that might resolve the update. But probably not the issue in general.

**Comment by selfarian** [comment]
Thank you for your help, creating the table worked. 
Just recognized, that the community version is discontinued. Args... need to think about, what to do, using now the latest community build and using tine just for my family ;).. so, we'll see :)

**Comment by lab-at-nohl** [comment]
> Just recognized, that the community version is discontinued. Args... need to think about, what to do, using now the latest community build and using tine just for my family ;).. so, we'll see :)

Don't worry. It will not affect you, continue with former business edition; Both version merged. Please read my full article.
https://github.com/orgs/tine-groupware/discussions/19

Just make sure that you will end up with **2023.11.x _after_** you have updated up to 2023.12.x (unusual path but works like a charm).
